### PR TITLE
update eraseOldFiles to not fail for ENOENT

### PR DESCRIPTION
**Pull request by mikephoran**
Tested it and it works.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -22,26 +22,35 @@ exports.url = function(url){
...
-        fs.unlinkSync(filePath);
+    if (file.indexOf('screenshot_') === 0) {
+      try {
+        var filePath = path.join(dirPath, 'file');
+        var stats = fs.statSync(filePath);
+        if (stats.isFile() && expireTime > stats.mtime.getTime()) {
+          console.log('Erasing file: ' + filePath + ' (older than ' + ttl + 'ms)');
+          fs.unlinkSync(filePath);
+        }  
+      }
```
Comment (Woodya) [inline lib/utils.js]: same line: `} catch`

File: lib/utils.js [hunk 2]
```diff
@@ -22,26 +22,35 @@ exports.url = function(url){
...
+      catch(e) {
+        if (e.code === 'ENOENT') {
+          console.log('Tried erasing file ' + filePath + ' but ENOENT thrown, skipping...')
+        } else {
+          throw e
```
Comment (Woodya) [inline lib/utils.js]: missing `;`, here and above. fix your pycharm lint settings.

File: lib/utils.js (hunk 1 above)
Comment (mikephoran) [inline lib/utils.js]: k fixed.

File: lib/utils.js [hunk 3]
```diff
@@ -22,26 +22,35 @@ exports.url = function(url){
 };
 
 /**
- * Delete all files in [dirPath] older than [ttl].  Does not modify subdirectories.
+ * Synchronously delete all files in [dirPath] older than [ttl].  Does not modify subdirectories.
  */
```
Comment (Woodya) [inline lib/utils.js:27]: remove the blank line between function comment and function definition.

ugh! nevermind. that's the convention in this codebase.

File: lib/utils.js (hunk 2 above)
Comment (mikephoran) [inline lib/utils.js]: k fixed.

File: lib/utils.js [hunk 4]
```diff
@@ -22,26 +22,35 @@ exports.url = function(url){
...
+          throw e
+        }
       }
     }
   })
 };
 
 /**
- * Delete file if it exists.
+ * Delete file asynchronously if it exists.
```
Comment (Woodya) [inline lib/utils.js:52]: if it's async. shouldn't you use `fs.unlink()` instead of `fs.unlinkSync()` ?

File: lib/utils.js (hunk 3 above)
Comment (mikephoran) [inline lib/utils.js:27]: haha yep.

**Comment by mikephoran** [comment]
@Woodya  Updated.  Let me know if that looks good!

**Comment by Woodya** [comment]
lgtm.

**Pull request merged**